pkg/registry/experimental/controller/etcd/etcd.go

```diff
@@ -88,6 +88,7 @@ func (r *ScaleREST) Update(ctx api.Context, obj runtime.Object) (runtime.Object,
 		return nil, false, errors.NewNotFound(extensions.Resource("replicationcontrollers/scale"), scale.Name)
 	}
 	rc.Spec.Replicas = scale.Spec.Replicas
+	rc.ResourceVersion = scale.ResourceVersion
```

Comment:
Ah, I'd forgotten that. That means if the client's intent was to only update the scale bit unconditionally, that means the server-side impl could stomp a change in another part of the rc/rs/deployment.

If the incoming scale has no ResourceVersion, I would expect the server to do the get/update inside a retryonconflict loop, fulfilling client intent while being a good citizen and not stomping bits unrelated to scale. I guess we don't do that today, so it's not a regression, but it would be an improvement (whether or not it happens for 1.2)